### PR TITLE
BIC-165 #Security doesn't show Login page by default

### PR DESCRIPTION
**Pull request by GeetaSajwan**
- setting login access true working
- logout working

### CODE REVIEW

File: src/bic/model-interaction.js [hunk 1]
```diff
@@ -224,27 +258,8 @@ The argument change event.
             reject();
           }
         } else {
-          model.set({interactionList: _.map(_.filter(app.interactions.models, function (value) {
-            return value.id !== window.BMP.BIC.siteVars.answerSpace.toLowerCase() && value.get('display') !== 'hide' && (!value.has('tags') || value.has('tags') && value.get('tags').length === 0 || _.filter(value.get('tags'), function (element) {
```
Comment (ashishtilara) [inline src/bic/model-interaction.js]: can't you use `app.siteVars`?

Comment (jokeyrhyme) [inline src/bic/model-interaction.js]: I believe this PR actually deletes those lines, anyway. There is another spot though where this might still apply...

Comment (ashishtilara) [inline src/bic/model-interaction.js]: Ooops yeah my bad, but seems like the code block is moved into `defaultView()` function

Comment (GeetaSajwan) [inline src/bic/model-interaction.js]: Yes, you are right Ashish. I have changed it to app.siteVars and it's working fine. Thanks.

Comment (jokeyrhyme) [inline src/bic/model-interaction.js]: @GeetaSajwan this change doesn't seem to have been pushed yet...

Comment (GeetaSajwan) [inline src/bic/model-interaction.js]: pushed the changes.

**Comment by jokeyrhyme** [comment]
LGTM :+1:

**Pull request merged**